src/main/java/mate/academy/dao/BookDao.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.Book;
+
+public interface BookDao {
+    Book create(Book book);
+
+    Optional<Book> findById(Long id);
+
+    List<Book> findAll();
+
+    Book update(Book book);
+
+    void deleteById(Long id);
```

Comment:
The `deleteById` method should return a boolean value instead of void. According to the task description, it should return `true` if a book was successfully deleted and `false` otherwise. Update the method signature to `boolean deleteById(Long id);`.